.travis.yml

```diff
@@ -0,0 +1,79 @@
+language: php
+
+jobs:
+  include:
+    - stage: test
+      php: "7.0"
+      env:
+        - TYPO3_VERSION=^8
+    - stage: test
+      php: "7.1"
+      env:
+        - TYPO3_VERSION=^8
+    - stage: test
+      php: "7.2"
+      env:
+        - TYPO3_VERSION=^8
+    - stage: test
+      php: "7.2"
+      env:
+        - TYPO3_VERSION=^9
+
```

Comment:
Please add a test for PHP 7.3 and TYPO3 v8 and v9

Reply:
Würde ich machen, wenn TYPO3 8 rausfliegt. Deal?